docs:#816 fix retention.local.target.bytes and retention.local.target.ms (#819)
redpanda-data/documentation: modules/reference/pages/rpk/rpk-cluster/rpk-cluster-storage-mount.adoc

= rpk cluster storage mount

Mount a topic to the Redpanda cluster from glossterm:tiered-storage[].

This command mounts a topic in the Redpanda cluster using log segments stored in Tiered Storage. 

> The topic may be optionally renamed with the `--to` flag.

Reviewer: ```suggestion
You can optionally rename the topic using the `--to` flag.
```

Requirements:

> - Tiered Storage must be enabled.

Reviewer: link to https://docs.redpanda.com/current/manage/tiered-storage/#enable-tiered-storage
